frontend/packages/console-dynamic-plugin-sdk/src/app/components/index.ts

```diff
@@ -0,0 +1 @@
+export * from './utils/resource-status';
```

Comment:
if `resource-status` is default export can just export that and shall we expose it from SDK as well?